DOKS-automatic-node-repair/README.md

```diff
@@ -1 +1,391 @@
+    11bdd0f1-8bd0-42dc-a3af-7a83bc319295    mobius-testing-cluster     lon1      1.21.2-do.2    false           running    basicnp
+    ```
+
+6. Get worker nodes status:
+
+    ```bash
+    kubectl get nodes
+    ```
+
+    The output looks similar to the following:
+    ```
+    NAME            STATUS   ROLES    AGE     VERSION
+    basicnp-8k4ep   Ready    <none>   2m52s   v1.21.2
+    basicnp-8k4es   Ready    <none>   2m16s   v1.21.2
+    ```
+    ### Note:
+
+      If for some reason the `kubectl` context was not set properly we can always fix it via:
+      
+      ```bash
+      doctl k8s cluster kubeconfig save "$CLUSTER_NAME"
+      ```
+
+## Digital Mobius
+
+### Overview
+
+[Digital Mobius](https://github.com/Qovery/digital-mobius) is an open source project written in the `Go` language which can automatically recycle unhealthy nodes from a `DigitalOcean Kubernetes` cluster (DOKS).
+
+### How it Works
+
+The logic inside the application is watching for nodes that are in an unhealthy state at a regular interval specified by the user. As described in the official `Kubernetes` documentation, a node is considered to be unhealthy if the [Node Condition](https://kubernetes.io/docs/concepts/architecture/nodes/#condition) is set to `Ready` and the status to `False` or `Unknown`. `Digital Mobius` looks for the `Unknown` status which is a result of the master node not being able to communicate with the worker node via the `kubelet`. It also provides a user configurable flag that specifies how much time has to pass before an unhealthy node needs to be recycled. If all the conditions enumerated previously are met then the affected node(s) will be re-created. It does so via a call to the DigitalOcean's Kubernetes [Delete Kubernetes Node](https://docs.digitalocean.com/reference/api/api-reference/#operation/delete_kubernetes_node) `REST API`.
+
+Below is a simplified diagram showing how `Digital Mobius` checks the worker node(s) state:
+
+![Digital Mobius Flow](content/img/digital-mobius-flow.png)
+
+### Configuration Overview
+
+In order for the application to work it needs a set of environment variables to be configured and available as seen in the [Helm Chart](https://github.com/Qovery/digital-mobius/blob/main/charts/Digital-Mobius/values.yaml) values file (explanations inline):
+
+```
+LOG_LEVEL: "info"
+DELAY_NODE_CREATION: "10m"                                  # a worker node gets recycled after being unhealthy for this period of time
+DIGITAL_OCEAN_TOKEN: "<your_digital_ocean_api_token>"       # personal DO API token value
+DIGITAL_OCEAN_CLUSTER_ID: "<your_digital_ocean_cluster_id>" # DOKS cluster ID that needs to be monitored
+```
+
+The most important values that we need to here provide is the DigitalOcean `API token`, the `Cluster ID` and the `Node Creation Delay` time interval which can be expressed in seconds or minutes using the appropriate suffix - e.g.: `10s`, `10m`, etc. 
```

Comment:
It would be nice to provide some guidance for users on how to choose the node creation delay time. The 10 minutes in the example seems like a good default, but I can see users wanting their nodes replaced right away and setting this to 0s or 10s, which would probably cause issues.

Reply:
Thanks and good catch! Added a **Note** describing why the default value of **10m** is used. The explanations for the `DRY RUN` mode added as well.